gallery/src/components/NxDrawer/NxDrawerEscExample.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import { NxDropdown, NxDrawer, NxButton, useToggle } from '@sonatype/react-shared-components';
+
+export default function NxDrawerEscExample() {
+  const [showDrawer, setShowDrawer] = useState(false);
+  const [dropdownIsOpen, onToggleDropdownCollapse] = useToggle(false);
+
+  const onClick = () => { alert('click'); };
+
+  return (
+    <>
+      <NxButton id="nx-drawer-esc-open-button" onClick={() => setShowDrawer(true)}>Open Drawer</NxButton>
+      {showDrawer && (
+        <NxDrawer id="nx-drawer-esc"
+                  onCancel={() => setShowDrawer(false)}
+                  headerTitle="An Example of NxDrawer with NxDropdown Nested Inside."
+                  aria-labelledby="nx-drawer-esc">
```

Comment:
By saying that it's labelled by itself, you're essentially saying that the text content of *all* children of the drawer should become its accessible name.  Probably not what we want.

Reply:
> By saying that it's labelled by itself, you're essentially saying that the text content of _all_ children of the drawer should become its accessible name. Probably not what we want.

Fixed here: 927586181ffbb6bf7d5f16ecd3ef12190cccc2eb